sandbox_externalkey_unix.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
-const udsBase = "/var/lib/docker/network/files/"
+const udsBase = "/run/libnetwork/"
```

Comment:
I think it would be better to keep it under docker hierarchy, so

`/run/docker/libnetwork`